content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,163 @@
+
+
+A more detailed explanation about this flow can also be found [here](https://www.ampproject.org/docs/reference/components/amp-access#login-flow). 
+
+## Add a comment
+
+<amp-img src="/static/img/comment.png" alt="Add comment" height="325" width="300"></amp-img>
+
+At this point, you can add a comment using the `amp-form` library. Notice how the presence of the form is conditioned by the `amp-access` component:
+
+[sourcecode:html]
+<form amp-access="loggedIn" amp-access-hide method="post" action-xhr="<%host%>/samples_templates/comment_section/submit-comment-xhr" target="_top">
+[/sourcecode]
+
+We specify a POST method and a XHR action, as non XHR actions are not allowed with POST methods in AMP.
+For sake of simplicity, we are not persisting comments, so it’s only possible to add a comment at the time; whenever a comment is added, the AMPByExample server replies with a json containing the entered text with some additions, like a timestamp, an avatar and a name for the user. 
+
+Find an example of response below:
+
+[sourcecode:json]
+{"Datetime":"09:34:21", 
+"User":"Charlie", 
+"Text":"Hello!", 
+"UserImg":"/img/ic_account_box_black_48dp_1x.png"}
+[/sourcecode]
+
+The form component will simply display those value inside the page using [amp-mustache](https://ampbyexample.com/components/amp-mustache/) template:
+
+[sourcecode:html]
+<div submit-success>
+<template type="amp-mustache">
+      	<div class="comment-user">
+              <amp-img width="44" class="user-avatar" height="44" alt="user" src="{{UserImg}}"></amp-img>
+              <div class="card comment">
+               <p><span class="user">{{User}}</span> <span class="date">{{Datetime}}</span></p>
+               <p>{{Text}}</p>
+              </div>
+            </div>
+      </template>
+</div>
+[/sourcecode]
+
+Notice how the template is wrapped in a div with the `submit-success` attribute. We are only checking if the value of the comment is not empty in this sample; in case it’s empty we return an error that would cause to execute the code:
+
+[sourcecode:html]
+<div submit-error>
+<template type="amp-mustache">
+            Error! Looks like something went wrong with your comment, please try to submit it again.
+      </template>
+</div>
+[/sourcecode]
+
+A little bit further on this, by using the attribute required, we are enforcing on the client side the presence of a text before hitting the Comment button:
+
+<amp-img src="/static/img/enforce-comment.png" alt="Enforce comment" height="325" width="300"></amp-img>
+
+[sourcecode:html]
+<input type="text" class="data-input" name="text" placeholder="Your comment..." required>
+[/sourcecode]
+
+Now add a comment and click comment, you should see something similar to the snapshot below:
+
+<amp-img src="/static/img/logout-button.png" alt="Comment added" height="352" width="300"></amp-img>
+
+## Logout
+Similar to the login button, the logout button is conditioned by the `amp-access` attribute:
```

Comment:
Same thing about conditioned. Maybe it's the right word and I'm just stupid though :)